symplyphysics/laws/electricity/circuits/varactor_junction_p_n_capacity.py

```diff
@@ -0,0 +1,41 @@
+from sympy import Eq, solve
+from symplyphysics import (units, Quantity, Symbol, validate_input, validate_output,
+                           dimensionless,)
+
+# Description
+## The principle of operation of the variator is based on the dependence of the barrier capacitance of the p-n junction on the voltage value.
+## Knowing the doping coefficient, the material parameter and the capacity without bias voltage, it is possible to calculate the capacity
+## of the varactor at a given voltage.
+
+## Law is: C =  C0 / (1 - V / V0)^y, where
+## C - barrier capacity of the p-n junction,
+## C0 - barrier capacitance of the p-n junction without bias voltage,
+## V - voltage,
+## V0 - material parameter,
+## y - doping coefficient.
+
+junction_capacitance = Symbol("junction_capacitance", units.capacitance)
+
+junction_transition_capacity_without_bias_voltage = Symbol("junction_transition_capacity_without_bias_voltage", units.capacitance)
```

Comment:
Name it according to similar parameter: `junction_capacitance_without_bias_voltage`